.github/workflows/muelu_clang_format.yml

```diff
@@ -0,0 +1,29 @@
+name: Check MueLu clang-format
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Update to version v4 to avoid the warning?

Reply:
let's do v3 since that's what the other workflows use for uniformity